src/index.css

```diff
@@ -10,6 +17,12 @@ body {
   margin: 0;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: rgb(140, 41, 241);
```

Comment:
Can we use `hex` color here?

Reply:
`#8c29f1` it's something like `accent-dark`

Reply:
Changed